src/utils/drawShapes.ts

```diff
@@ -20,22 +17,104 @@ export const drawSquare = (
   centerY: number,
   s?: number
```

Comment:
What do you think about making these parameters with defaults?

```suggestion
  size: number = 4
```

Doesn't matter much at all, but might make reading the function signature slightly nicer.

Reply:
that is better, thanks